fsw/nos3_defs/targets.cmake

```diff
@@ -75,21 +75,17 @@ SET(MISSION_NAME "NOS3")
 # should be an integer.
 SET(SPACECRAFT_ID 42)
 
-# UI_INSTALL_SUBDIR indicates where the UI data files (included in some apps) should
-# be copied during the install process.
-SET(UI_INSTALL_SUBDIR "host/ui")
-
-# FT_INSTALL_SUBDIR indicates where the black box test data files (lua scripts) should
-# be copied during the install process.
-SET(FT_INSTALL_SUBDIR "host/functional-test")
-
-# Application List
-SET(APPLICATION_LIST
+# The "MISSION_GLOBAL_APPLIST" is a set of apps/libs that will be built
+# for every defined target.  These are built as dynamic modules
+# and must be loaded explicitly via startup script or command.
+# This list is effectively appended to every TGTx_APPLIST in targets.cmake.
+# Example:
+list(APPEND MISSION_GLOBAL_APPLIST
     #
     # Libraries
     #
-        cfs_lib
-        cryptolib
+        #cfs_lib
```

Comment:
Suggestion to remove CS, MD, MM, SBN targets since they are not included as submodules to avoid confusion.

Reply:
Should be resolved in commit https://github.com/nasa/nos3/pull/203/commits/026654a70621127d4a2b448637878ad7c9143652